### ISSUE TITLE
MsgExplorer throws exception when processing msi files

### ISSUE THREAD

**Issue by Jmcleodfoss** [Jmcleodfoss/msgreader#5]
As described in a comment in Issue #2, the following exception is thrown when processing an MSI file (which is a Compound Binary File formatted file).

```
java.lang.NullPointerException
        at io.github.jmcleodfoss.msg.Directory.getChildren(Directory.java:115)
        at io.github.jmcleodfoss.msg.NamedProperties.<init>(NamedProperties.java:50)
        at io.github.jmcleodfoss.msg.MSG.<init>(MSG.java:57)
```

I assume this is because the (undocumented) MSI file does not include a Named Properties section - at least, the Named Properties Mapping Entry is null.

**Comment by Jmcleodfoss** [comment]
Should be addressed by ddafa748bf3378520624da5488209849501bca23. @ebourg, are you able to check this? It works okay for me on the test MSI file you provided.

**Comment by Jmcleodfoss** [comment]
Released today in version 1.0.3.

**Comment by ebourg** [comment]
I confirm it works, thank you.
